docs/source/enable_cc_lifecycle.md

```diff
@@ -0,0 +1,300 @@
+have the advantage of being scriptable, making it suitable for proposing
+configuration updates to a large number of channels, and is the recommended
+process for editing a channel configuration.
+
+Note that the `enable_lifecycle.json` uses sample values, for example
+`org1Policies` and the `msp_identifier`, which will be specific to your deployment):
+
+```
+{
+  "org1Policies": {
+      "Endorsement": {
+           "mod_policy": "Admins",
+           "policy": {
+               "type": 1,
+               "value": {
+               "identities": [
+                  {
+	                 "principal": {
+	           	         "msp_identifier": "Org1ExampleCom",
+	           	         "role": "PEER"
+	                 },
+	                 "principal_classification": "ROLE"
+	              }
+              	],
+              	"rule": {
+                  "n_out_of": {
+			           "n": 1,
+			           "rules": [
+			           	{
+			           		"signed_by": 0
+			           	}
+			           ]
+			       }
+              	},
+              	"version": 0
+              }
+           },
+           "version": "0"
+      }
+   },
+  "org2Policies": {
+      "Endorsement": {
+           "mod_policy": "Admins",
+           "policy": {
+               "type": 1,
+               "value": {
+               "identities": [
+                  {
+	                 "principal": {
+	           	         "msp_identifier": "Org2ExampleCom",
+	           	         "role": "PEER"
+	                 },
+	                 "principal_classification": "ROLE"
+	              }
+              	],
+              	"rule": {
+                  "n_out_of": {
+			           "n": 1,
+			           "rules": [
+			           	{
+			           		"signed_by": 0
+			           	}
+			           ]
+			       }
+              	},
+              	"version": 0
+              }
+           },
+           "version": "0"
+      }
+   },
+   "appPolicies": {
+ 		"Endorsement": {
+			"mod_policy": "Admins",
+			"policy": {
+				"type": 3,
+				"value": {
+					"rule": "MAJORITY",
+					"sub_policy": "Endorsement"
+				}
+			},
+			"version": "0"
+		},
+		"LifecycleEndorsement": {
+			"mod_policy": "Admins",
+			"policy": {
+				"type": 3,
+				"value": {
+					"rule": "MAJORITY",
+					"sub_policy": "Endorsement"
+				}
+			},
+			"version": "0"
+		}
+   },
+   "acls": {
+		"_lifecycle/CommitChaincodeDefinition": {
+			"policy_ref": "/Channel/Application/Writers"
+		},
+		"_lifecycle/QueryChaincodeDefinition": {
+			"policy_ref": "/Channel/Application/Readers"
+		},
+		"_lifecycle/QueryNamespaceDefinitions": {
+			"policy_ref": "/Channel/Application/Readers"
+		}
+   }
+}
+```
+
+## Edit the channel configurations
+
+### System channel updates
+
+Because configuration changes to the system channel only involve parameters
+inside the configuration of the individual ordering service organizations within
+the channel configuration, the relevant organizations can make the edits without
```

Comment:
"orderering service organizations" should be "non-orderer organizations". Also, can we simplify the sentence to "Because configuration changes to the system channel only involve the individual non-orderer organizations within the channel configuration, ..."?

Reply:
Good catch! I fixed this all up to reflect the fact that the orderer org has to suggest the change but the peer org has to sign it.